src/rpc/rpcevo.cpp

```diff
@@ -242,7 +242,7 @@ void protx_register_fund_help()
             "                         It has to match the private key which is later used when operating the masternode.\n"
             "5. \"votingAddress\"       (string, required) The voting key address. The private key does not have to be known by your wallet.\n"
             "                         It has to match the private key which is later used when voting on proposals.\n"
-            "                         If set to \"0\" or an empty string, ownerAddr will be used.\n"
+            "                         If set an empty string, ownerAddr will be used.\n"
```

Comment:
Looks like I missed this one in #2618 🙈 , let's fix it here maybe

```suggestion
            "                         If set to an empty string, ownerAddress will be used.\n"
```

Also brings back the missing `to `.